904820a523154e2435d16efbc923da59ece8a95e..4f8dfdba3df0401bdca7ceab1c0fd031446b53c6
7483a38d4c4eb823362f093a3e721335b5854365..3c16e4a7d0c9dfcd62dbb42e37c2cf564d6c578b

From da21c305164c3e585e29e20242afc5a31f91989f Mon Sep 17 00:00:00 2001
From: Michael Cronenworth <mike@cchtml.com>
Date: Thu, 23 Jan 2020 17:05:27 -0600
Subject: [PATCH] wbemprox: Global variable compatibility update for gcc 10.

Signed-off-by: Michael Cronenworth <mike@cchtml.com>
Signed-off-by: Alexandre Julliard <julliard@winehq.org>
---
 dlls/wbemprox/main.c             | 2 ++
 dlls/wbemprox/wbemprox_private.h | 4 ++--
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/dlls/wbemprox/main.c b/dlls/wbemprox/main.c
index e6ccd5c21c0..0fc11caa6ad 100644
--- a/dlls/wbemprox/main.c
+++ b/dlls/wbemprox/main.c
@@ -38,4 +38,6 @@ WINE_DEFAULT_DEBUG_CHANNEL(wbemprox);
 static HINSTANCE instance;
 
+struct list *table_list;
+
 typedef HRESULT (*fnCreateInstance)( LPVOID *ppObj );
 
diff --git a/dlls/wbemprox/wbemprox_private.h b/dlls/wbemprox/wbemprox_private.h
index d52affa82af..b3cd58db4ff 100644
--- a/dlls/wbemprox/wbemprox_private.h
+++ b/dlls/wbemprox/wbemprox_private.h
@@ -22,6 +22,6 @@
 #include "wine/unicode.h"
 
-IClientSecurity client_security DECLSPEC_HIDDEN;
-struct list *table_list DECLSPEC_HIDDEN;
+extern IClientSecurity client_security DECLSPEC_HIDDEN;
+extern struct list *table_list DECLSPEC_HIDDEN;
 
 enum param_direction
--
